src/boot/directives.js

```diff
@@ -26,7 +30,7 @@ export default ({ app }) => {
     // called when the bound element's parent component
     // and all its children are mounted.
     mounted () {
```

Comment:
Remove altogether if we want to remove the console logs, along with other lifecycle hook methods.